coalib/bearlib/languages/definitions/shell.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class shell:
+    aliases = 'bash', 'sh'
+
+    extensions = '.sh', '.bash', '.zsh'
+    comment_delimiter = '#'
+    multiline_comment_delimiters = {': '': '''}
+    string_delimiters = {'"': '"', "'": "'"}
+    multiline_string_delimiters = string_delimiters
+    encapsulators = {'[': ']', '{': '}'}
+    keywords = [
+        'if', 'then', 'else', 'elif', 'fi', 'case', 'coproc', 'esac',
+        'for', 'select', 'while', 'until', 'do', 'done', 'in',
+        'function', 'time']
```

Comment:
http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html

Should we just cover the POSIX shell spectrum?